packages/grid/_modules_/grid/hooks/features/pagination/gridPaginationSelector.ts

```diff
@@ -2,15 +2,21 @@ import { createSelector } from 'reselect';
 import { GridState } from '../core/gridState';
 import { visibleSortedGridRowIdsSelector } from '../filter/gridFilterSelector';
 import { GridPaginationState } from './gridPaginationState';
+import { gridContainerSizesSelector } from "../../root/gridContainerSizesSelector";
 
 export const gridPaginationSelector = (state: GridState): GridPaginationState => state.pagination;
 
 export const gridPaginatedVisibleSortedGridRowIdsSelector = createSelector(
   gridPaginationSelector,
   visibleSortedGridRowIdsSelector,
-  (pagination, visibleSortedRows) => {
+    gridContainerSizesSelector,
+    (pagination, visibleSortedRows, containerSizes) => {
+      if (!containerSizes) {
+          return []
```

Comment:
I can throw if you prefer
The selector is never used in a render, but if it was I prefer not to have a buggy behavior like `containerSizes!.virtualRowsCount`